mbuild/utils/decorators.py

```diff
@@ -0,0 +1,16 @@
+"""Some helpful decorators"""
+
+
+from warnings import warn
+
+
+def deprecated(warning_string=None):
+    def old_function(fcn):
+        def wrapper(*args, **kwargs):
+            printed_message = '{0} is deprecated. '.format(fcn.__name__)
+            if warning_string:
```

Comment:
You could just make the default for warning string be empty string to remove the conditional